libs/go/aea_end2end/fetchai_private_key.txt

```diff
@@ -0,0 +1 @@
+6f975c0d18150ca8af593abd721c094d1ed07b58a5ed01eba448c58adcb56b72
```

Comment:
Can we not generate this on the fly each time?

Reply:
probably added by mistake